modules/microshift-olm-deploy-ops-global-ns.adoc

```diff
@@ -0,0 +1,198 @@
+
+[NOTE]
+====
+If you want to use an Operator installed in a different namespace, or in more than one namespace, both the catalog source and the Subscription CR that references the Operator must be running in the `openshift-marketplace` namespace.
+====
+
+.Prerequisites
+* The {oc-first} is installed.
+* Operator Lifecycle Manager (OLM) is installed.
+* You have created a custom catalog in the global namespace.
+
+.Procedure
+
+. Confirm that OLM is running by using the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager get pod -l app=olm-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                            READY   STATUS    RESTARTS   AGE
+olm-operator-66c97f894f-55wp2   1/1     Running   0          126m
+----
+
+. Confirm that the OLM catalog Operator is running by using the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager get pod -l app=catalog-operator
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                READY   STATUS    RESTARTS   AGE
+catalog-operator-7cf77c459d-jbrd7   1/1     Running   0          126m
+----
+
+[NOTE]
+====
+The following steps assume you are using the global namespace, `openshift-marketplace`. The catalog must run in the same namespace as the Operator. The Operator must support the *All namespaces* mode.
+====
+
+. Create the `CatalogSource` object by using the following example YAML:
++
+.Example catalog source YAML
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: operatorhubio-catalog
+  namespace: openshift-marketplace <1>
+spec:
+  sourceType: grpc
+  image: quay.io/operatorhubio/catalog:latest
+  displayName: Community Operators <2>
+  publisher: OperatorHub.io
+  grpcPodConfig:
+    securityContextConfig: restricted <3>
+  updateStrategy:
+    registryPoll:
+      interval: 60m
+----
+<1> The global namespace. Setting the `metadata.namespace` to `openshift-marketplace` enables the catalog to run in all namespaces. Subscriptions in any namespace can reference catalogs created in the `openshift-marketplace` namespace.
+<2> Community Operators are not installed by default with OLM for {miroshift-short}. Listed here for example only.
+<3> The value of `securityContextConfig` must be set to `restricted` for {microshift-short}.
+
+. Apply the `CatalogSource` configuration by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f _<my-catalog-source.yaml>_ <1>
+----
+<1> Replace _<my-catalog-source.yaml>_ with your catalog source configuration file name.
+
+. To verify that the catalog source is applied, check for `READY` state by using the following command:
++
+[source,terminal]
+----
+$ oc describe catalogsources.operators.coreos.com -n openshift-operators operatorhubio-catalog
```

Comment:
@ShaunaDiaz the `openshift-operators` should be `openshift-marketplace` because https://github.com/openshift/openshift-docs/pull/70128/files#diff-3e486f2a15729abbe0165210eff361e2811b98290c6656804ed6cea4d0ea5e6eR65

its output should also be updated.

if you want to keep it, need to change https://github.com/openshift/openshift-docs/pull/70128/files#diff-3e486f2a15729abbe0165210eff361e2811b98290c6656804ed6cea4d0ea5e6eR65 , https://github.com/openshift/openshift-docs/pull/70128/files#diff-3e486f2a15729abbe0165210eff361e2811b98290c6656804ed6cea4d0ea5e6eR53 and https://github.com/openshift/openshift-docs/pull/70128/files#diff-3e486f2a15729abbe0165210eff361e2811b98290c6656804ed6cea4d0ea5e6eR163